vault/config.go

```diff
@@ -147,6 +147,7 @@ type ProfileSection struct {
 	STSRegionalEndpoints    string `ini:"sts_regional_endpoints,omitempty"`
 	SessionTags             string `ini:"session_tags,omitempty"`
 	TransitiveSessionTags   string `ini:"transitive_session_tags,omitempty"`
+	FederationSessionTags   string `ini:"federation_session_tags,omitempty"`
```

Comment:
Is there a case for. re-using `session_tags` instead of introducing `federation_session_tags`?

Reply:
We could re-use session_tags. I think the reason I went this route was because when initially investigated this feature, I made an assumption that the session tags were used for the federation token, but it wasnt, and it became unclear where they were used. I opted to be more explicit about where it's being used (context of federation). I would be ok with reusing SessionTags